charts/homebridge/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+appVersion: 3.1.0
+version: 1.0.0
+name: homebridge
+description: A lightweight NodeJS server that emulates the iOS HomeKit API
+type: application
+keywords:
+  - homebridge
+  - homekit
+home: https://homebridge.io/
```

Comment:
For `home:` Can you please use `https://github.com/k8s-at-home/charts/tree/master/charts/homebridge` and put `https://homebridge.io/` under `sources:`?